cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -902,6 +909,34 @@ protected Future<Void> clusterId(KafkaStatus kafkaStatus) {
                 });
     }
 
+    /**
+     * Configures the default users quota in Kafka in case that the {@link QuotasPluginKafka} is used
+     *
+     * @return  Future which completes when the default quotas are configured
+     */
+    protected Future<Void> defaultQuotas() {
+        QuotasPlugin quotas = kafka.quotas();
+
+        if (quotas instanceof QuotasPluginKafka quotasPluginKafka) {
+            LOGGER.debugCr(reconciliation, "Attempt to alter default user quotas in Kafka");
+            String bootstrapHostname = KafkaResources.bootstrapServiceName(reconciliation.name()) + "." + reconciliation.namespace() + ".svc:" + KafkaCluster.REPLICATION_PORT;
+            LOGGER.debugCr(reconciliation, "Creating AdminClient for setting default quota using {}", bootstrapHostname);
+
+            try (Admin kafkaAdmin = adminClientProvider.createAdminClient(bootstrapHostname, this.coTlsPemIdentity.pemTrustSet(), this.coTlsPemIdentity.pemAuthIdentity())) {
+                List<ClientQuotaAlteration.Op> ops = QuotasPluginUtils.getListOfClientQuotaAlternationOps(quotasPluginKafka);
+
+                // when the `null` is set here, it will configure the default quota values for all users
+                ClientQuotaAlteration clientQuotaAlteration = new ClientQuotaAlteration(new ClientQuotaEntity(Collections.singletonMap(ClientQuotaEntity.USER, null)), ops);
+                kafkaAdmin.alterClientQuotas(List.of(clientQuotaAlteration));
+            } catch (KafkaException e) {
+                LOGGER.warnCr(reconciliation, "Kafka exception configuring default quotas {}", e.getMessage());
+                return Future.failedFuture(new Throwable("Failed to configure default user quotas"));
+            }
+        }
```

Comment:
This does not seem to have any unit tests. It also might make sense to have it in some utils class for easier testing.

Reply:
Moved to `QuotasPluginUtils` and added UTs, thanks :)